src/styles.css

```diff
@@ -1 +1,4 @@
 /* You can add global styles to this file, and also import other style files */
```

Comment:
change the angular configuration.
use scss

Reply:
ok

Reply:
done thanks!